xml/System.Text.Json/JsonValueKind.xml

```diff
@@ -74,7 +74,7 @@
       </ReturnValue>
       <MemberValue>7</MemberValue>
       <Docs>
-        <summary>Indicates that a value is the JSON value <see langword="null" />.</summary>
+        <summary>The JSON value <see langword="null" />.</summary>
```

Comment:
These shouldn't be langwords (like discussed previously). Same with true/false. They are JSON-specific literals. I think we decided on marking them bold instead (if I recall correctly).